### ISSUE TITLE
Include summaries in export

### ISSUE THREAD

**Issue by bloodykheeng** [pxlrbt/filament-excel#146]
i have a table with summaries at the bottom but i acnt bulk export the table with those summaries 
Inthat when i click bulk select the row for summaries isnt among the selectable ones
Below are some of the columns ive defined with summaries

TextColumn::make('sku')
    ->summarize(Range::make()->limit(5))

  TextColumn::make('group_total')
                    ->label('Total # of Group Members')
                    ->sortable()
                    ->numeric(
                        decimalPlaces: 0,
                        decimalSeparator: '.',
                        thousandsSeparator: ',',
                    )
                    ->searchable()->toggleable(isToggledHiddenByDefault: false)
                    ->summarize(Sum::make()->numeric()->label("")),

 TextColumn::make('female_total')
                    ->label('# of Female members')
                    ->numeric(
                        decimalPlaces: 0,
                        decimalSeparator: '.',
                        thousandsSeparator: ',',
                    )
                    ->sortable()->searchable()->toggleable(isToggledHiddenByDefault: false)
                    ->summarize(Sum::make()->numeric()->label("")),

 TextColumn::make('below_35_total')
                    ->label('Total # of members below 35 years')
                    ->numeric(
                        decimalPlaces: 0,
                        decimalSeparator: '.',
                        thousandsSeparator: ',',
                    )
                    ->sortable()->searchable()->toggleable(isToggledHiddenByDefault: false)
                    ->summarize(Sum::make()->numeric()->label("")),

                TextColumn::make('on_time_total')
                    ->label('# of on-time payments')
                    ->numeric(
                        decimalPlaces: 0,
                        decimalSeparator: '.',
                        thousandsSeparator: ',',
                    )
                    ->sortable()->searchable()->toggleable(isToggledHiddenByDefault: false)
                    ->summarize(Sum::make()->numeric()->label("")),
TextColumn::make('surplus_total')
                    ->label('Surplus/Deficit')
                    ->numeric(
                        decimalPlaces: 0,
                        decimalSeparator: '.',
                        thousandsSeparator: ',',
                    )
                    ->sortable()->searchable()->toggleable(isToggledHiddenByDefault: false)
                    ->summarize(Sum::make()->numeric()->label("")),

                TextColumn::make('opd_total')
                    ->label('# Member OPD')
                    ->numeric(
                        decimalPlaces: 0,
                        decimalSeparator: '.',
                        thousandsSeparator: ',',
                    )
                    ->sortable()->searchable()->toggleable(isToggledHiddenByDefault: false)
                    ->summarize(Sum::make()->numeric()->label("")),

                TextColumn::make('ipd_total')
                    ->label('# Member IPD')
                    ->numeric(
                        decimalPlaces: 0,
                        decimalSeparator: '.',
                        thousandsSeparator: ',',
                    )
                    ->sortable()->searchable()->toggleable(isToggledHiddenByDefault: false)
                    ->summarize(Sum::make()->numeric()->label("")),

                TextColumn::make('percent_ontime_total')
                    ->label('% paid ontime')
                    ->numeric(
                        decimalPlaces: 0,
                        decimalSeparator: '.',
                        thousandsSeparator: ',',
                    )
                    ->suffix('%')
                    ->sortable()->searchable()->toggleable(isToggledHiddenByDefault: false),
                // ->summarize(Average::make()->numeric()->label("")),

                TextColumn::make('f_anc_attendance_total')
                    ->label('Total 4th ANC visits')
                    ->numeric(
                        decimalPlaces: 0,
                        decimalSeparator: '.',
                        thousandsSeparator: ',',
                    )
                    ->sortable()->searchable()->toggleable(isToggledHiddenByDefault: false)
                    ->summarize(Sum::make()->numeric()->label("")),

                TextColumn::make('f_anc_school_total')
                    ->label('# 4th ANC school visits')
                    ->numeric(
                        decimalPlaces: 0,
                        decimalSeparator: '.',
                        thousandsSeparator: ',',
                    )
                    ->sortable()->searchable()->toggleable(isToggledHiddenByDefault: false)
                    ->summarize(Sum::make()->numeric()->label("")),

am using fillament v3

Any one to help

**Comment by pxlrbt** [comment]
There is just no support for it currently.

**Comment by bloodykheeng** [comment]
Any suggestions or ideas on how i can archieve it as i wait for future updates

**Comment by pxlrbt** [comment]
Not really, sorry. It's probably easiest to create a custom export via the underlying `maatswebiste/laravel-excel` package

**Comment by bloodykheeng** [comment]
ok thanks lemi hope in future you add that.  thaks fir the support though

**Comment by zindazed** [comment]
Would this be added in any time soon? seems important

**Comment by pxlrbt** [comment]
@zindazed I am not using Summaries anywhere in my projects, so I haven't implemented this yet. I am open for a clean PR though.

**Comment by thetahayasin** [comment]
facing same issue summaries not in export no workaround found
